libs/sdk-core/src/models.rs

```diff
@@ -629,6 +629,7 @@ pub struct NodeState {
     pub max_single_payment_amount_msat: u64,
     pub max_chan_reserve_msats: u64,
     pub connected_peers: Vec<String>,
+    pub max_receivable_single_payment_amount_msat: u64,
```

Comment:
I would add some rustdocs about the meaning of the changed `NodeState` fields, for example

```rust
pub struct NodeState {
    ...

    /// Maximum receivable in a single payment (single channel)
    pub max_receivable_single_payment_amount_msat: u64,

    /// Total receivable on all available channels
    pub inbound_liquidity_msats: u64,
}
```

Reply:
Added the proposed docs in [e2d48a4](https://github.com/breez/breez-sdk/pull/1062/commits/e2d48a4f6a2f640877ea4b10c24fce1d4c8b019a)